assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss

```diff
@@ -37,4 +37,28 @@
 			padding: $grid-gap-desktop 0 0 0;
 		}
 	}
+
+	.googlesitekit-wizard-progress__content {
+		margin: 0 auto;
+		max-width: 465px;
```

Comment:
Just curious, why did you choose this maximum width specifically? Feels a bit like an arbitrary value, maybe this should be in a variable somewhere?

Reply:
@felixarntz So the max width is as per the provided designs. If you look at the dashboard setup here: https://www.figma.com/file/f07vatCPHzubuJoXkgd5RU/SiteKit-Phase-1_-Onboarding?node-id=1098%3A17677 , the content is centered with a max width of 465px. I replaced it by a variable though. Let me know if that works for you or do you think we should get rid of the max width or shall we follow the design closely?